### PR TITLE
Add new authors to 1.5 contributors list

### LINKED ISSUE #287: Update Contributors for 1.5
We have lots of new folks who have contributed to 1.5.  :tada:  Below is the list of folks who should be added to the "Current Authors and Collaborators" list in 1.5:

Megan Grodowitz
Md. Wasi-ur- Rahman
Min Si
Akhil Langer

We should also move some of the previous contributors to the alumni section.

### PR DESCRIPTION
**Pull request by RaymondMichael**
This fixes #287

### CODE REVIEW

**Comment by agrippa** [comment]
@RaymondMichael @jdinan sorry I didn't see issue 287 earlier -- would it also be appropriate to change my affiliation to Georgia Tech in this change?

**Comment by RaymondMichael** [comment]
Done

File: content/coverpage.tex [hunk 1]
```diff
@@ -52,20 +52,19 @@ \section*{Current Authors and Collaborators}
 \item Swen Boehm, \ac{ORNL}
 \item Aurelien Bouteiller, \ac{UTK}
 \item Barbara Chapman, \ac{SBU}
-\item Robert Cernohous, Cray Inc.
+\item Robert Cernohous, \ac{HPE}
```
Comment (naveen-rn) [inline content/coverpage.tex]: I forgot to say this during the last release - it is Bob Cernohous

File: content/coverpage.tex [hunk 2]
```diff
@@ -52,20 +52,19 @@ \section*{Current Authors and Collaborators}
...
 \item James Culhane, \ac{LANL}
 \item Tony Curtis, \ac{SBU}
 \item James Dinan, Intel
 \item Mike Dubman, Mellanox
-\item Karl Feind, \ac{HPE}
 \item Manjunath Gorentla Venkata, \ac{ORNL}
```
Comment (jdinan) [inline content/coverpage.tex]: Please add @anshumang: Anshuman Goswami, NVIDIA

Comment (nspark) [inline content/coverpage.tex]:
```suggestion
\item Manjunath Gorentla Venkata, \ac{Mellanox}
```

**Pull request merged**